plugins/css-blank-pseudo/test/_browser.mjs

```diff
@@ -50,7 +50,7 @@ import { promises as fsp } from 'fs';
 
 	if (!process.env.DEBUG) {
 		const browser = await puppeteer.launch({
-			headless: true,
+			headless: 'new',
```

Comment:
>   Puppeteer old Headless deprecation warning:
    In the near feature `headless: true` will default to the new Headless mode
    for Chrome instead of the old Headless implementation. For more
    information, please see https://developer.chrome.com/articles/new-headless/.
    Consider opting in early by passing `headless: "new"` to `puppeteer.launch()`
    If you encounter any bugs, please report them to https://github.com/puppeteer/puppeteer/issues/new/choose.

Reply:
This was already added on main when updating dependencies.